backend/src/main/java/ai/verta/modeldb/authservice/RoleServiceUtils.java

```diff
@@ -511,18 +511,14 @@ private void setRoleBindingOnAuthService(boolean retry, RoleBinding roleBinding)
               GetCollaboratorResponseItem.newBuilder()
                   .setAuthzEntityType(collaborator.getAuthzEntityType())
                   .setVertaId(collaborator.getId());
-          getCollaboratorResponseBuilder
-              .getPermissionBuilder()
-              .setCollaboratorType(CollaboratorType.READ_ONLY);
+          CollaboratorPermissions.Builder collPermBuilder = CollaboratorPermissions.newBuilder();
```

Comment:
I'm pretty sure I have already reviewed all these changes in another PR.

Reply:
Yes, it is the same.

Reply:
Did we copy code then? I couldn't review this file because it was too close to the other one, so I wouldn't be able to perceive the differences.